src/background.js

```diff
@@ -42,7 +45,7 @@ async function spinWith(func, ...args) {
 }
 
 async function processSelectedMessages(folder, operation="move", goToFolder=false) {
-  let { markAsRead } = await browser.storage.local.get({ markAsRead: true });
+  let { markAsRead, notificationActive, operationCounters } = await browser.storage.local.get({ markAsRead: true, notificationActive: DEFAULT_PREFERENCES.notificationActive, operationCounters: DEFAULT_PREFERENCES.operationCounters });
```

Comment:
While you are changing this could you also have it take `markAsRead` from DEFAULT_PREFERENCES ?

Reply:
Done.